src/main/java/org/terasology/structureTemplates/internal/systems/AddItemsToChestSystem.java

```diff
@@ -64,6 +71,12 @@ public void onSpawnStructureEvent(StructureBlocksSpawnedEvent event, EntityRef e
         }
     }
 
+    /**
+     * A private method used by @link onSpawnStructureEvent. It actually adds the items to the chest.
+     * @param chest The chest to add the items to.
+     * @param item The item to be added.
+     * @param blockFactory If this item is a block, this parameter is used to make a new instance of the block as an item.
+     */
```

Comment:
That the method is private, is nothing that should be mentioned in a javadoc since it is redundant. The `@link` syntax won't work as you miss the brackets `{` and `}`.

Also the fact which method is using a method is not something worth documenting. This information can quickly can become out of date when someone just calls that method too. Also IDEs can show you the call hierachy for methods. So if someone wants to know it he can use the IDE feature.

Most IDEs show the javadoc of a method in a formatted way when you hover a call of that method with the mouse.

So while I find it great that you proposed an addition of javadoc I think in those cases we are better off without javadoc.